boards/arm/thingy53_nrf5340/pm_static_thingy53_nrf5340_cpuapp_ns.yml

```diff
@@ -8,16 +8,16 @@ mcuboot_pad:
   size: 0x200
 tfm_secure:
   address: 0x10000
-  size: 0xc200
+  size: 0xc000
   span: [mcuboot_pad, tfm]
 tfm_nonsecure:
-  address: 0x1c200
+  address: 0x1c000
   size: 0xd3e00
   span: [app]
 tfm:
   address: 0x10200
   region: flash_primary
-  size: 0xc000
+  size: 0xbe00
 app:
   address: 0x1c200
```

Comment:
Should the `app` be moved to `0x1c000` and size needs to be aligned with `tfm_nonsecure`?

Reply:
Good point change it